pkgs/tools/misc/tea/default.nix

```
@@ -0,0 +1,21 @@
+{ lib, buildGoModule, fetchgit }:
+
+buildGoModule rec {
+  pname = "tea";
+  version = "v0.7.0";
```

The numerical representation is enough.

```suggestion
  version = "0.7.0";
```